packages/components/src/components/select-native/select-native.jsx

```diff
@@ -62,6 +62,7 @@ const SelectNative = ({
     use_text,
     value,
     data_testid,
+    hide_top_placeholder = false,
```

Comment:
should we describe default value here as well as in defaultProps on the line 207?

Reply:
Will remove default prop, since while migrating we wont be using them